package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "preinstall": "npx ensure-node-env",
     "emulator:build": "./gradlew build",
-    "emulator:instrumentation": "./gradlew Backpack:connectedDebugAndroidTest -PlocalTests=true",
+    "emulator:instrumentation": "./gradlew Backpack:connectedDebugAndroidTest backpack-compose:connectedDebugAndroidTest -PlocalTests=true",
```

Comment:
same as on CI - run compose tests as well. this was the easiest way to do it - we just always run the full test suite no matter what gets published